windows-nsis/openvpn.nsi

```diff
@@ -344,11 +412,11 @@ Function CoreSetup
 	!insertmacro WriteRegStringIfUndef HKLM "SOFTWARE\${PACKAGE_NAME}" "priority"    "NORMAL_PRIORITY_CLASS"
 	!insertmacro WriteRegStringIfUndef HKLM "SOFTWARE\${PACKAGE_NAME}" "log_append"  "0"
 
-	; install openvpnserv.exe as a services
 	DetailPrint "Installing OpenVPN Interactive Service..."
-	nsExec::ExecToLog /OEM '"$INSTDIR\bin\openvpnserv.exe" -install'
+	SimpleSC::InstallService "OpenVPNServiceInteractive" "OpenVPN Interactive Service" "32" "3" "$INSTDIR\bin\openvpnserv.exe" "tap0901/dhcp" "" ""
```

Comment:
Yes, we can do that. SimpleSC does not produce any output on error, so this is harmless.